### PR TITLE
[Student][MBL-14066] Find Studio for submission type with lots of LTIs

### PR DESCRIPTION
**Pull request by CalvinKern**

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1025842&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: libs/canvas-api-2/src/main/java/com/instructure/canvasapi2/apis/ExternalToolAPI.kt [hunk 1]
```diff
@@ -29,8 +29,10 @@ import retrofit2.http.Url
 
 
 internal object ExternalToolAPI {
+    // This returns a paged response, so either we have to depaginate, or pull down 100, since we're typically looking
+    // for the Studio LTI.
     internal interface ExternalToolInterface {
-        @GET("{contextId}/external_tools?include_parents=true")
+        @GET("{contextId}/external_tools?include_parents=true&per_page=100")
```
Comment (Blackchai) [inline libs/canvas-api-2/src/main/java/com/instructure/canvasapi2/apis/ExternalToolAPI.kt]: woohoo!

**Comment by JordanMarshall** [comment]
I realize this is merged, but the established pattern for specifying the `per_page` query parameter is to set `usePerPageQueryParam = true` in the constructor for the RestParam object, which is typically constructed in the manager class.

**Pull request merged**